Data/Scripts/GardenConquest/Messaging/RequestProcessor.cs

```diff
@@ -171,10 +171,12 @@ public class RequestProcessor {
 				// Build text based on whether or not player is in faction
 				switch (owner.OwnerType) {
 					case GridOwner.OWNER_TYPE.FACTION:
-						errorMessage = "Your faction does not have control of that ship's Main Cockpit!";
+						errorMessage = "Your faction does not have control of that ship's ";
+						errorMessage += (ConquestSettings.getInstance().CommandsRequireClassifier == true ? "Hull Classifier!" : "Main Cockpit!");
```

Comment:
This should be unnecessary in the case of simple ownership, because nothing will be on the the fleet list that they don't already have control of.